ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -61,6 +61,14 @@ export default class AdvancedTab extends PureComponent {
     showLedgerTransportWarning: false,
   };
 
+  showTestNetworksRef = React.createRef();
+
+  componentDidMount() {
+    if (window.location.hash.match(/show-testnets/u)) {
+      this.showTestNetworksRef.current.scrollIntoView({ behavior: 'smooth' });
```

Comment:
We should probably focus this element as well, for keyboard users

Reply:
ah, nice catch